source/Irrlicht/CMakeLists.txt

```diff
@@ -44,19 +38,21 @@ if(OGLES1_ENABLED)
 	find_library(EGL_LIBRARY NAMES EGL REQUIRED)
 
 	message(STATUS "Found OpenGLES: ${OPENGLES_LIBRARY}")
-	link_libraries(${OPENGLES_LIBRARY} ${EGL_LIBRARY})
+	link_libraries("${OPENGLES_LIBRARY}" "${EGL_LIBRARY}")
 endif()
 check_symbol_exists(_IRR_COMPILE_WITH_OGLES2_ "IrrCompileConfig.h" OGLES2_ENABLED)
 if(OGLES2_ENABLED)
 	find_package(OpenGLES2 REQUIRED)
-	include_directories(${OPENGLES2_INCLUDE_DIR} ${EGL_INCLUDE_DIR})
+
+	include_directories("${OPENGLES2_INCLUDE_DIR}", "${EGL_INCLUDE_DIR}")
```

Comment:
warning on this line, the comma sign `,` shouldn't be there I think
```
CMake Warning (dev) at lib/irrlichtmt/source/Irrlicht/CMakeLists.txt:47:
  Syntax Warning in cmake code at column 48

  Argument not separated from preceding token by whitespace.
This warning is for project developers.  Use -Wno-dev to suppress it.
```

Reply:
Correct. Thank you for noticing this, it would likely cause a compilation error!